tests/tests_app/cli/test_cd.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from lightning.app.cli.commands.cd import _CD_FILE, cd
+from lightning.app.cli.commands.pwd import pwd
+
+
+def test_cd():
```

Comment:
So with `lightning cd` you can change into directories that don't exist?

Reply:
Yes, for now. I need to add an endpoint to validate if a directory exists.

Reply:
I have added a TODO in the code for it.